translations/translate.js

```diff
@@ -36,7 +36,7 @@ translate = {
         // build a more efficient lookup
         var lookup = {}
 
-        for (var r in one2one)
+        for (var r = 0, fLen = one2one.length; r < fLen; r++)
```

Comment:
if you need order preserved when looping this array you could use `one2one.forEach( function(element) {} )`

Reply:
Done.

Reply:
And, added quite a few more of these.